### PR TITLE
[Teleop] Add TargetBearing arrow

### PR DESCRIPTION
**Pull request by CameronTressler**
The target bearing is represented by the smaller of the two arrows. The larger blue arrow is still the tangent arrow based on GPS bearing. Also logs and displays during log playback. @bli76  do you want to improve the graphics a bit??

And yes, I did make the main rover arrow smaller. (It's on the list to reduce code duplication for logging, I know it's awful)

image

### CODE REVIEW

**Review by rbridges12 — APPROVED** [review]
looks good

**Review by bli76 — APPROVED** [review]
The newline in the lcm struct isn't showing up on git but everything else looks good

File: base_station/gui/src/store/modules/autonomy.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ const state = {
   playbackOdomLat: [],
   playbackOdomLon: [],
   playbackOdomBearing: [],
-  playbackGpsBearing: []
+  playbackGpsBearing: [],
```
Comment (cgiger00) [inline base_station/gui/src/store/modules/autonomy.js:16]: mention in readme this is what the tangent arrow is?

**Review by cgiger00 — COMMENTED** [review]
lgtm, just add smth to clarify what the tangent bearing is

**Pull request merged**